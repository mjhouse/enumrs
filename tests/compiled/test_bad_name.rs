use enumrs::Tagged;

#[derive(Tagged)]
pub enum TestEnum {
    // Tag names should be valid identifiers
    // in line with rust ident rules, which
    // means they can't begin with a number.
    #[tag(1padding)]
    Variant = 1,
}

pub fn main() {}