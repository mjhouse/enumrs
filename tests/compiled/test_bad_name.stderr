error: Invalid tag name: '1padding'
 --> tests/compiled/test_bad_name.rs:8:11
  |
8 |     #[tag(1padding)]
  |           ^^^^^^^^
